core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/impl/PathPropertiesImpl.kt

```diff
@@ -255,3 +253,33 @@ fun getOffsetOfTrackLocationOnChunksOrThrow(
 ): Distance {
     return getOffsetOfTrackLocationOnChunks(infra, location, chunks) ?: throw RuntimeException()
 }
+
+@JvmName("buildChunkPath")
```

Comment:
Same, a comment about the fact the the returned path is only a subset of the given chunks would be nice